src/dl_node.cpp

```diff
@@ -182,10 +182,10 @@ Status DLNode::fetchResults(BlobMap& outputs) {
                 const auto blob = infer_request.GetBlob(realModelOutputName);
                 SPDLOG_DEBUG("[Node: {}] Creating copy of blob from model:{}, inferRequestStreamId:{}, blobName:{}",
                     getName(), modelName, streamId.value(), realModelOutputName);
-                const auto copiedBlob = blobClone(blob);
-                if (copiedBlob == nullptr) {
-                    SPDLOG_ERROR("[Node: {}] Cannot copy blob - buffer sizes mismatch", getName());
-                    return StatusCode::INTERNAL_ERROR;
+                InferenceEngine::Blob::Ptr copiedBlob = nullptr;
```

Comment:
Minor; 
= nullptr; unnecessary

Additionally you removed SPDLOG_ERROR message